src/gpt2/transformer.rs

```diff
@@ -37,17 +35,14 @@ impl MLP {
 
         let c_fc = GPTConv1D::new(p / "c_fc", config.n_embd * 4, config.n_embd);
         let c_proj = GPTConv1D::new(p / "c_proj", config.n_embd, config.n_embd * 4);
-        let activation = Box::new(match &config.afn {
+        let activation = match &config.afn {
             Some(activation_enum) => match activation_enum {
-                Activation::gelu => _gelu_new,
-                Activation::relu => _relu,
-                Activation::swish => _swish,
-                Activation::gelu_new => _gelu_new,
-                Activation::mish => _mish,
-                Activation::tanh => _tanh,
+                Activation::gelu => &Activation::gelu_new,
```

Comment:
BTW @guillaume-be this mapping from `gelu` to `gelu_new` is intentional, right?

Reply:
Yes - see https://github.com/huggingface/transformers/blob/84caa23301f2f25dfca0737198ff26c3d711ed63/src/transformers/modeling_openai.py#L142

Reply:
Alright! :) Just wanted to check back as it puzzled me.